plasmapy/__init__.py

```diff
@@ -15,5 +15,5 @@
 import sys
 import warnings
 
-if sys.version_info[:2] < (3, 6):  # coveralls: ignore
+if sys.version_info[:2] < (3, 6):  # pragma: no cover
```

Comment:
You should pull in changes from upstream so this doesn't undo all my hard work :P